server-spi-private/src/main/java/org/keycloak/models/utils/DefaultRequiredActions.java

```diff
@@ -84,6 +84,17 @@ public static void addActions(RealmModel realm) {
         }
 
         addUpdateLocaleAction(realm);
+
+        if (realm.getRequiredActionProviderByAlias("delete_account") == null) {
```

Comment:
This also needs to be added to existing realms during migration

Reply:
you mean the delete_account action -- makes sens